src/ServiceControl.Persistence/ITrialLicenseMetadataProvider.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace ServiceControl.Persistence
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public interface ITrialLicenseMetadataProvider
```

Comment:
Nitpicking: metadata is the data about data, right? So shouldn't this be just `ITrialLicenseDataProvider`? In other words, I don't see the _meta_ aspect here.